acquisition-event-producer/src/main/scala/com/gu/acquisition/services/DefaultAcquisitionService.scala

```diff
@@ -27,12 +27,10 @@ class DefaultAcquisitionService(config: DefaultAcquisitionServiceConfig)(implici
   override def submit[A: AcquisitionSubmissionBuilder](a: A)(implicit ec: ExecutionContext) = {
     val ov = ophanService.submit(a).value
     val gv = gAService.submit(a).value
-    val kv = kinesisService.submit(a).value
```

Comment:
We can go further and actually remove the kinesisService from the `acquisition-event-producer`.
This module is tied to the thrift model. We can later re-implement the kinesis writing logic elsewhere

Reply:
Okay - I've removed that now!